### DISCUSSION TITLE
Platform User roles to allow or disable project creation

### DISCUSSION THREAD

**Discussion by JMcGenn** [langfuse/langfuse#1858] [Ideas]
### Describe the feature or potential improvement

Hi, 

I am impressed with LangFuse's functionality so far.

I can't see a way of controlling users ability to create projects. I would like to see platform level roles like platform admins that can choose users who can and can't create new projects at a platform creator/ view only roles. I am wondering if this is something on the roadmap or could be added as a feature in the future. 

Thanks, 
Joe

**Comment by marcklingen** [comment]
Great idea, has not been a priority yet. I think the idea here would be to default to users only having access to projects they were invited to or offer some additional admin capabilities on self-hosted instances.

**Reply by JMcGenn** [reply]
Thanks for the reply. I agree, defaulting users to invitations on project and having platform admins would allow improved governance and maintenance of this as a platform in some use cases for organisations.

**Reply by superpoussin22** [reply]
agree that we need to disable project creation cause everyone who has access to langfuse can create projects and the only way to see if a project is created is to parse the tables in the database :(

**Comment by marcklingen** [comment]
Organizations added this, only org owners and admins can create new projects: https://langfuse.com/changelog/2024-08-13-organizations
